lib/gooddata/lcm/actions/base_action.rb

```diff
@@ -18,6 +18,11 @@ class BaseAction
       class << self
         include Dsl::Dsl
 
+        RUNNING_STATUS_PARAM = 'running_status'.to_sym
```

Comment:
Please rename to ABORT_ON_ERROR to understandable

Reply:
Done update code